### PR TITLE
Display min and max lending rate for the upcoming days and the target lending rate.

### PR DESCRIPTION
**Pull request by Natim**
image

### CODE REVIEW

File: src/Bps2Taeg.elm [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    let
+        initialModel =
+            { installmentsCount = Just 10
+            , bps = Just 609
+            , minPaymentPlan = []
+            , maxPaymentPlan = []
+            }
+
+        ( minPaymentPlan, maxPaymentPlan ) =
+            getMinMaxPaymentPlan initialModel
```
Comment (PascalLeMerrer) [inline src/Bps2Taeg.elm:52]: get + sthg is a good function name in an OO language
In FP the function is equivalent to a value, so we usually don't use verbs in function names

```suggestion
            minMaxPaymentPlan initialModel
```

File: src/Bps2Taeg.elm [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    in
+    ( { newModel | minPaymentPlan = minPaymentPlan, maxPaymentPlan = maxPaymentPlan }, Cmd.none )
+
+
+getMinMaxPaymentPlan : Model -> ( List Days.Installment, List Days.Installment )
+getMinMaxPaymentPlan model =
+    case
+        ( model.installmentsCount, model.bps )
+    of
+        ( Just installmentsCount, Just bps ) ->
```
Comment (PascalLeMerrer) [inline src/Bps2Taeg.elm:81]: This function is really long, and not easy to read.
Could you extract things from the let in clause?

File: src/Bps2Taeg.elm [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+                _ ->
+                    ( [], [] )
+
+        _ ->
+            ( [], [] )
+
+
+viewInstallment : Int -> Days.Installment -> Html Msg
+viewInstallment i installment =
+    tr [ class "" ]
```
Comment (PascalLeMerrer) [inline src/Bps2Taeg.elm:148]:
```suggestion
    tr []
```

File: src/Bps2Taeg.elm [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+                        , th []
+                            [ text feeTitle ]
+                        ]
+                    ]
+                , List.indexedMap viewInstallment paymentPlan |> tbody []
+                ]
+
+
+annual_interest_rate : List Days.Installment -> String
+annual_interest_rate paymentPlan =
```
Comment (PascalLeMerrer) [inline src/Bps2Taeg.elm:197]:
```suggestion
annualInterestRate PaymentPlan =
```

File: src/Bps2Taeg.elm [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+        Just taeg ->
+            Round.round 2 (taeg * 100)
+
+        Nothing ->
+            "-,--"
+
+
+view : Model -> Html Msg
+view { bps, installmentsCount, minPaymentPlan, maxPaymentPlan } =
+    div [ class "row" ]
```
Comment (PascalLeMerrer) [inline src/Bps2Taeg.elm:212]: I really prefer for the view code to be splitted in several functions. 
It allows to name parts of the generated DOM in order to express your intents, and makes it easier to identify them for a newcomer.
However it's really a matter of preference, so it's up to you.

File: src/Bps2Taeg.elm (hunk 2 above)
Comment (k4nar) [reply src/Bps2Taeg.elm:81]: Maybe it would be easier to have one function for the min and one for the max?

**Review by PascalLeMerrer — APPROVED** [review]

**Pull request merged**